src/components/AddForm/AddForm.jsx

```diff
@@ -21,6 +26,22 @@ function AddForm({
     const [inputValue, setInputValue] = useState('');
     const [deadline, setDeadline] = useState('');
     const [validMsg, setValidMsg] = useState('');
```

Comment:
bad naming.
validMsg means your message is valid but in reality it contains error message